fix: Can't find `@nuxt/utils` when using `nuxt-edge`

**🐛 The bug**

Hey 👋🏻
When using `nuxt-composition-api` together with `nuxt-edge`, the module breaks because `@nuxt/utils` can't be found (because it's called `@nuxt/utils-edge` then).

`   │   Error: Cannot find module '@nuxt/utils'                                                        │`

**🛠️ To reproduce**
Steps to reproduce the behavior:

1. Create a new project and cd into it `yarn create nuxt-app rm-composition-api-test-edge && cd rm-composition-api-test-edge` 
2. Replace Nuxt with Nuxt Edge `yarn remove nuxt && yarn add nuxt-edge`
3. Add the module `yarn add @nuxtjs/composition-api`
4. Build! `yarn dev`

**🌈 Expected behaviour**
A wrapper that detects nuxt-edge and loads the corresponding package. IIRC we have such a thing in a few modules alreaady.

--- a/src/index.ts
+++ b/src/index.ts
@@ -4,8 +4,17 @@ import { readdirSync, copyFileSync, existsSync, mkdirpSync } from 'fs-extra'
 import type { Module } from '@nuxt/types'
 import normalize from 'normalize-path'
 
-// eslint-disable-next-line
-const utils = require('@nuxt/utils')
+const loadUtils = () => {
+  try {
+    // Try to load nuxt edge utils first
+    return require('@nuxt/utils-edge')
+  } catch {
+    // if it fails, fall back to normal nuxt utils
+    return require('@nuxt/utils')
+  }
+}
+
+const utils = loadUtils()
 
 const compositionApiModule: Module<any> = function () {
   const libRoot = resolve(__dirname, '..')

